release/nightly_tests/stress_tests/test_parallel_tasks_memory_pressure.py

```diff
@@ -19,10 +19,23 @@ def allocate_memory(
     return 1
 
 
+def get_container_memory_limit():
+    with open("/sys/fs/cgroup/memory.max") as f:
+        limit_bytes = int(f.read().strip())
+    return limit_bytes
+
+
+def get_container_memory_usage():
+    with open("/sys/fs/cgroup/memory.current") as f:
+        usage_bytes = int(f.read().strip())
+    return usage_bytes
```

Comment:
We have existing `get_system_memory()` and `get_used_memory()` that works for cgroup. You can just call those.

Reply:
Nice! Let me refactor it